command/handlers/deleteTrackHandler.js

```diff
@@ -9,19 +12,18 @@ class DeleteAlbumHandler {
 
     handle(unqfy, trackData) {
         if(this.validateData(trackData)){
-            unqfy.deleteTrackFrom(trackData.artistName, trackData.albumName)
+            let deletedTrack= unqfy.deleteTrackFrom(trackData.artistName, trackData.albumName, trackData.trackName);
+            console.log(deletedTrack.name + " fue borrado exitosamente");
         }
     }
        
-    validateData(data) {
-        let hasData = Boolean(data);
-        let hasCompleteData = Boolean((data || {}).artistName) && Boolean((data || {}).trackName);
-    
-        if (!hasData || !hasCompleteData) {
-            console.log(`No se pudo completar la operación. Faltan datos: ${(data || {}).artistName } ${(data|| {}).tracktName}`);
-        }
+    validateData(trackData) {
+        let validator = new Validator(trackData);
```

Comment:
No te parece mejor hacerlo como en los otros? Validar y continuar? Sin agregar este validateData que devuelve un true pelado al final. Podríamos en todo caso devolver `validator.isValidFor(...)`

Reply:
Para refactor. No me di cuenta que copie otra cosa y me daba que no existia el hasData.

Reply:
ok for my